CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Removed
+
+- The `inotify-tools` as dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852) 
```

Comment:
Revert this CHANGELOG change and replace with a fragment file in `changelog/fragments`